config/install/tripal.tripalfield_collection.trpcultivate_experiments.yml

```diff
@@ -917,18 +833,796 @@ fields:
+        cardinality: -1
+        required: false
+        storage_settings:
+            storage_plugin_id: chado_storage
+            storage_plugin_settings:
+                base_table: project
+                prop_table: projectprop
+        settings:
+            termIdSpace: local
+            termAccession: 'project_germcollection'
+        display:
+          view:
+            default:
+              region: content
+              label: above
+              weight: 20
+          form:
+            default:
+              region: content
+              weight: 20
+
+    -   name: ghouse_featureofinterest
+        content_type: greenhouse_experiment
+        label: Feature of Interest
+        type: chado_property_type_default
+        description: List the names of the core phenotypic traits you are interested in. These are usually very focused on your experimental question. If you are exploring shattering then examples can be "Shattered Seed Weight' or 'Shattering Scale".
+        cardinality: -1
+        required: false
+        storage_settings:
+            storage_plugin_id: chado_storage
+            storage_plugin_settings:
+                base_table: project
+                prop_table: projectprop
+        settings:
+            termIdSpace: SOSA
+            termAccession: FeatureOfInterest
+        display:
+          view:
+            default:
+              region: content
+              label: above
+              weight: 20
+          form:
+            default:
+              region: content
+              weight: 20
+
+    -   name: ghouse_objectives
+        content_type: greenhouse_experiment
+        label: Objectives
+        type: chado_property_type_default
+        description: The objectives of this experiment. These should be the questions you are trying to answer with this experiment. For example, "Determine the magnitude of effect environment has on my trait of interest.
+        cardinality: -1
+        required: false
+        storage_settings:
+            storage_plugin_id: chado_storage
+            storage_plugin_settings:
+                base_table: project
+                prop_table: projectprop
+        settings:
+            termIdSpace: SIO
+            termAccession: '000337'
+        display:
+          view:
+            default:
+              region: content
+              label: above
+              weight: 20
+          form:
+            default:
+              region: content
+              weight: 20
+
+    -   name: ghouse_growingmedia
+        content_type: greenhouse_experiment
+        label: Growing Media
+        type: chado_property_type_default
+        description: Use for the substrates used for growing plants in containers; for the artificial media or substrates used for growing organisms in vitro USE culture media. Materials offering adequate rooting and support conditions for maintaining plant cultivation; for cell or tissue culture use.
```

Comment:
The wording here feels very confusing to me. To me "Use for the substrates..." implies this field is maybe optional? The 2 sentences as a whole don't really provide guidance to the user. The random capitalizing of USE also throws me off a bit 😆 
You'll also want to address this in this field for growth chamber experiment!

Reply:
How about this "Record the growing media being used in this controlled environment experiment." 🙂

Reply:
Yes that works. 👍 It would be nice to provide a couple of examples if possible, though I admit I'm not sure how easy that information is for us to find 😅